addons/website/static/src/snippets/s_image_gallery/000.js

```diff
@@ -182,6 +189,10 @@ const GallerySliderWidget = publicWidget.Widget.extend({
         });
         this.$carousel.on('slid.bs.carousel.gallery_slider', update);
 
+        if (!this.editableMode) {
```

Comment:
Instead of fixing here we should fix it in options.js, you can override method ```cleanForSave``` and remove class there.

Reply:
Definitely, Let me look into it ,Thanks for the suggestion! :)